src/libs/oaVerify.ts

```diff
@@ -0,0 +1,22 @@
+import { isValid, verificationBuilder, openAttestationVerifiers } from "@govtechsg/oa-verify";
+import { ethers } from "ethers";
+import { NETWORK_NAME, INFURA_API_KEY } from "../config";
+
+const provider = new ethers.providers.FallbackProvider(
+  [
+    { priority: 1, provider: new ethers.providers.InfuraProvider(NETWORK_NAME, INFURA_API_KEY) },
+    // { priority: 10, provider: new ethers.providers.AlchemyProvider(NETWORK_NAME, ALCHEMY_API_KEY) },
+  ],
+  1
+);
+
+let verify;
+const getVerifier = () => {
```

Comment:
so it will rely on the default oa-verify cache for the moment right? What's your plan / idea?

Reply:
Yes. So far, it seems quite reliable.

Cache miss: ~4 to 5 seconds
Cache hit: Less than ~80 to 150 ms

Reply:
But the cache is only 5 mins? Shouldn't we find a way to increase it ?

Reply:
Perhaps we can modify the `oa-verify` library to allow for a custom cache period to be specified?

Reply:
https://www.npmjs.com/package/did-resolver

There is a cache option